chart/hyrax/values.yaml

```diff
@@ -238,6 +238,144 @@ postgresql:
   # persistence:
   #   size: 10Gi
 
+
+## Nginx proxy is used to keep puma from having to serve static assets
+## and to act as an auth proxy for Cantelope
+nginx:
+  enabled: false
+  # The set up below does malicious bot / ip blocking and mounts
+  # vaolumes to allow nginx to server assets and other public directory items
+  # image:
+  #   registry: registry.gitlab.com
+  #   repository: notch8/scripts/bitnami-nginx
+  #   tag: 1.21.5-debian-10-r4
+  # extraVolumes:
+  #   - name: "uploads"
+  #     persistentVolumeClaim:
+  #       claimName: dev-nnp-uploads
```

Comment:
good point. it needs to match the uploadsVolume name. I wonder if for example purposes the sub out of `{{ .Values.global.hyraxHostName }}-uploads` is a reasonable thing to use - you have to have something from global that's specific otherwise.